lua/neotest-go/init.lua

```diff
@@ -161,6 +161,12 @@ function adapter.build_spec(args)
   if fn.isdirectory(position.path) ~= 1 then
     location = fn.fnamemodify(position.path, ":h")
   end
+
+  local run_flag = {}
+  if position.type == "test" then
+    run_flag = { "--run", "\\^" .. utils.get_prefix(args.tree, position.name) .. "\\$" }
```

Comment:
All other arguments use just one dash. Not sure what the difference would be here (if any) but perhaps keep it consistent?

Also, why the backslashes?

```suggestion
    run_flag = { "-run", "^" .. utils.get_prefix(args.tree, position.name) .. "$" }
```

Reply:
Those slashes are from [here](https://github.com/nvim-neotest/neotest-go/pull/72/files#diff-ba39acc61170d012d9fe7694e11f1e9c7060a687fda264dec455706a38da4a8aL165),  without it works also fine

Reply:
Not sure what @sergii4 says about them, but I think we don't need those backslashes. And I'd vote for `-run` (one dash) to keep the arguments consistently using one dash.